common/lib/dependabot/pull_request_creator/labeler.rb

```diff
@@ -1,32 +1,54 @@
-# typed: true
+# typed: strict
```

Comment:
Just checking if this is intentionally strict and not strong like some of the other recently-typed files.

Reply:
Good catch, you're right. Thanks! I'll fix it in the morning.

Reply:
@bdragon I had a look, and I can't use strong because of the use of `T.unsafe` 😢

I'll have to come back to this once I properly type all the HTTP clients and can get rid of the `T.unsafe`.